util/logtools/encounter_tools.ts

```diff
@@ -158,7 +158,7 @@ export class EncounterFinder {
       // Therefore we can safely initialize everything.
       if (this.currentFight.startTime)
         this.onEndFight(line, cZ, 'Zone Change');
-      if (this.currentZone.zoneName)
+      if (this.currentZone.zoneName !== undefined && this.currentZone.zoneName.length > 0)
```

Comment:
I don't think we should special case `zoneName`.  It comes from the log line.

```suggestion
      if (this.currentZone.zoneName !== undefined)
```